lib/ui/components/cv_password_field.dart

```diff
@@ -44,8 +50,10 @@ class _CVPasswordFieldState extends State<CVPasswordField> {
           ),
           labelText: 'Password',
         ),
+        textInputAction: TextInputAction.done,
         validator: widget.validator,
         onSaved: widget.onSaved,
+        focusNode: widget.focus_node,
```

Comment:
It's better to keep focusNode as the first parameter because it kind of defines the text field